mkdocs/docs/experiments/concepts/chaos-resources/chaos-engine/application-details.md

```diff
@@ -165,13 +165,12 @@ spec:
   engineState: "active"
   annotationCheck: "false"
   # provide the comma separated auxiliary applications details
-  auxiliaryAppInfo: "app=nginx:nginx,app=busybox:default"
+  auxiliaryAppInfo: "app=nginx:nginx,app=default:busybox"
   appinfo:
     appns: "default"
     applabel: "app=nginx"
     appkind: "deployment"
-  chaosServiceAccount: pod-delete-sa
```

Comment:
remove appinfo section

Reply:
also add TARGET_NODE env

Reply:
The spec section is not added in any other manifest also in this.

Reply:
because we were using pod-delete experiment earlier - which doesn't have any mandatory ENV unlike node-drain